gamemodes/terrortown/gamemode/client/cl_hudpickup.lua

```diff
@@ -57,26 +46,17 @@ function GM:HUDItemPickedUp(itemname)
 	-- source's localisation
 	pickup.name = "#" .. itemname
 	pickup.holdtime = 5
-	pickup.font = "DefaultBold"
 	pickup.fadein = 0.04
 	pickup.fadeout = 0.3
 	pickup.color = Color(255, 255, 255, 255)
-	pickup.upper = false
-
-	surface.SetFont(pickup.font)
 
-	local w, h = surface.GetTextSize(pickup.name)
-
-	pickup.height = h
-	pickup.width = w
-
-	if self.PickupHistoryLast >= pickup.time then
-		pickup.time = self.PickupHistoryLast + 0.05
+	if PICKUP.last >= pickup.time then
+		pickup.time = PICKUP.last + 0.05
 	end
 
-	table.insert(self.PickupHistory, pickup)
+	table.insert(PICKUP.items, pickup)
 
-	self.PickupHistoryLast = pickup.time
+	PICKUP.last = pickup.time
```

Comment:
Correct me if i am wrong, but isn't this the same code as in HUDWeaponPickup? If so i would suggest putting that in one function, to reduce redundancy, if there is no good reason for it to be standalone.